### PR TITLE
[MAINTENANCE] Define JSON Schemas for ColumnAggregate Expectations

### PR DESCRIPTION
**Pull request by josectobar**
Define pydantic models and generate JSON schemas for ColumnAggregate Expectations
- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: great_expectations/expectations/core/expect_column_unique_value_count_to_be_between.py [hunk 1]
```diff
@@ -43,6 +44,12 @@
 )
 MIN_VALUE_DESCRIPTION = "The minimum number of unique values allowed."
 MAX_VALUE_DESCRIPTION = "The maximum number of unique values allowed."
+STRICT_MIN_DESCRIPTION = (
+    "If True, the column must have strictly more unique value count than min_value to pass."
+)
+STRICT_MAX_DESCRIPTION = (
+    "If True, the column must have strictly fewer unique value count than max_value to pass."
+)
```
Comment (josectobar) [inline great_expectations/expectations/core/expect_column_unique_value_count_to_be_between.py:52]: I added these missing descriptions.

File: great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py [hunk 2]
```diff
@@ -130,6 +132,10 @@ class ExpectColumnKLDivergenceToBeLessThan(ColumnAggregateExpectation):
            {TAIL_WEIGHT_HOLDOUT_DESCRIPTION}
         bucketize_data (boolean): \
             {BUCKETIZE_DATA_DESCRIPTION}
+        min_value (float or None): \
+            {MIN_VALUE_DESCRIPTION}
+        max_value (float or None): \
+            {MAX_VALUE_DESCRIPTION}
```
Comment (josectobar) [inline great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py:138]: Same situation here, these descriptions were missing.

File: great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py [hunk 3]
```diff
@@ -187,6 +192,29 @@ class ExpectColumnDistinctValuesToContainSet(ColumnAggregateExpectation):
...
+                "const": model._library_metadata,
+            }
+            schema["properties"]["short_description"] = {
+                "type": "string",
+                "const": EXPECTATION_SHORT_DESCRIPTION,
+            }
+            schema["properties"]["supported_data_sources"] = {
+                "type": "array",
+                "const": SUPPORTED_DATA_SOURCES,
+            }
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_distinct_values_to_contain_set.py]:
```suggestion
            schema["properties"]["metadata"]["properties"].update(
                {
                    "data_quality_issues": {
                        "title": "Data Quality Issues",
                        "type": "array",
                        "const": DATA_QUALITY_ISSUES,
                    },
                    "library_metadata": {
                        "title": "Library Metadata",
                        "type": "object",
                        "const": model._library_metadata,
                    },
                    "short_description": {
                        "title": "Short Description",
                        "type": "string",
                        "const": EXPECTATION_SHORT_DESCRIPTION,
                    },
                    "supported_data_sources": {
                        "title": "Supported Data Sources",
                        "type": "array",
                        "const": SUPPORTED_DATA_SOURCES,
                    },
                }
            )
```

File: great_expectations/expectations/core/expect_column_distinct_values_to_equal_set.py [hunk 4]
```diff
@@ -186,6 +191,29 @@ class ExpectColumnDistinctValuesToEqualSet(ColumnAggregateExpectation):
...
+                "const": model._library_metadata,
+            }
+            schema["properties"]["short_description"] = {
+                "type": "string",
+                "const": EXPECTATION_SHORT_DESCRIPTION,
+            }
+            schema["properties"]["supported_data_sources"] = {
+                "type": "array",
+                "const": SUPPORTED_DATA_SOURCES,
+            }
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_distinct_values_to_equal_set.py]:
```suggestion
            schema["properties"]["metadata"]["properties"].update(
                {
                    "data_quality_issues": {
                        "title": "Data Quality Issues",
                        "type": "array",
                        "const": DATA_QUALITY_ISSUES,
                    },
                    "library_metadata": {
                        "title": "Library Metadata",
                        "type": "object",
                        "const": model._library_metadata,
                    },
                    "short_description": {
                        "title": "Short Description",
                        "type": "string",
                        "const": EXPECTATION_SHORT_DESCRIPTION,
                    },
                    "supported_data_sources": {
                        "title": "Supported Data Sources",
                        "type": "array",
                        "const": SUPPORTED_DATA_SOURCES,
                    },
                }
            )
```

File: great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py [hunk 5]
```diff
@@ -334,6 +350,29 @@ class ExpectColumnKLDivergenceToBeLessThan(ColumnAggregateExpectation):
...
+                "const": model._library_metadata,
+            }
+            schema["properties"]["short_description"] = {
+                "type": "string",
+                "const": EXPECTATION_SHORT_DESCRIPTION,
+            }
+            schema["properties"]["supported_data_sources"] = {
+                "type": "array",
+                "const": SUPPORTED_DATA_SOURCES,
+            }
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_kl_divergence_to_be_less_than.py]:
```suggestion
            schema["properties"]["metadata"]["properties"].update(
                {
                    "data_quality_issues": {
                        "title": "Data Quality Issues",
                        "type": "array",
                        "const": DATA_QUALITY_ISSUES,
                    },
                    "library_metadata": {
                        "title": "Library Metadata",
                        "type": "object",
                        "const": model._library_metadata,
                    },
                    "short_description": {
                        "title": "Short Description",
                        "type": "string",
                        "const": EXPECTATION_SHORT_DESCRIPTION,
                    },
                    "supported_data_sources": {
                        "title": "Supported Data Sources",
                        "type": "array",
                        "const": SUPPORTED_DATA_SOURCES,
                    },
                }
            )
```

File: great_expectations/expectations/core/expect_column_most_common_value_to_be_in_set.py [hunk 6]
```diff
@@ -176,6 +184,29 @@ class ExpectColumnMostCommonValueToBeInSet(ColumnAggregateExpectation):
...
+                "const": model._library_metadata,
+            }
+            schema["properties"]["short_description"] = {
+                "type": "string",
+                "const": EXPECTATION_SHORT_DESCRIPTION,
+            }
+            schema["properties"]["supported_data_sources"] = {
+                "type": "array",
+                "const": SUPPORTED_DATA_SOURCES,
+            }
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_most_common_value_to_be_in_set.py]:
```suggestion
            schema["properties"]["metadata"]["properties"].update(
                {
                    "data_quality_issues": {
                        "title": "Data Quality Issues",
                        "type": "array",
                        "const": DATA_QUALITY_ISSUES,
                    },
                    "library_metadata": {
                        "title": "Library Metadata",
                        "type": "object",
                        "const": model._library_metadata,
                    },
                    "short_description": {
                        "title": "Short Description",
                        "type": "string",
                        "const": EXPECTATION_SHORT_DESCRIPTION,
                    },
                    "supported_data_sources": {
                        "title": "Supported Data Sources",
                        "type": "array",
                        "const": SUPPORTED_DATA_SOURCES,
                    },
                }
            )
```

File: great_expectations/expectations/core/expect_column_proportion_of_unique_values_to_be_between.py [hunk 7]
```diff
@@ -203,6 +210,29 @@ class ExpectColumnProportionOfUniqueValuesToBeBetween(ColumnAggregateExpectation
...
+                "const": model._library_metadata,
+            }
+            schema["properties"]["short_description"] = {
+                "type": "string",
+                "const": EXPECTATION_SHORT_DESCRIPTION,
+            }
+            schema["properties"]["supported_data_sources"] = {
+                "type": "array",
+                "const": SUPPORTED_DATA_SOURCES,
+            }
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_proportion_of_unique_values_to_be_between.py]:
```suggestion
            schema["properties"]["metadata"]["properties"].update(
                {
                    "data_quality_issues": {
                        "title": "Data Quality Issues",
                        "type": "array",
                        "const": DATA_QUALITY_ISSUES,
                    },
                    "library_metadata": {
                        "title": "Library Metadata",
                        "type": "object",
                        "const": model._library_metadata,
                    },
                    "short_description": {
                        "title": "Short Description",
                        "type": "string",
                        "const": EXPECTATION_SHORT_DESCRIPTION,
                    },
                    "supported_data_sources": {
                        "title": "Supported Data Sources",
                        "type": "array",
                        "const": SUPPORTED_DATA_SOURCES,
                    },
                }
            )
```

File: great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py [hunk 8]
```diff
@@ -254,6 +259,29 @@ class ExpectColumnQuantileValuesToBeBetween(ColumnAggregateExpectation):
...
+                "const": model._library_metadata,
+            }
+            schema["properties"]["short_description"] = {
+                "type": "string",
+                "const": EXPECTATION_SHORT_DESCRIPTION,
+            }
+            schema["properties"]["supported_data_sources"] = {
+                "type": "array",
+                "const": SUPPORTED_DATA_SOURCES,
+            }
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py]:
```suggestion
            schema["properties"]["metadata"]["properties"].update(
                {
                    "data_quality_issues": {
                        "title": "Data Quality Issues",
                        "type": "array",
                        "const": DATA_QUALITY_ISSUES,
                    },
                    "library_metadata": {
                        "title": "Library Metadata",
                        "type": "object",
                        "const": model._library_metadata,
                    },
                    "short_description": {
                        "title": "Short Description",
                        "type": "string",
                        "const": EXPECTATION_SHORT_DESCRIPTION,
                    },
                    "supported_data_sources": {
                        "title": "Supported Data Sources",
                        "type": "array",
                        "const": SUPPORTED_DATA_SOURCES,
                    },
                }
            )
```

File: great_expectations/expectations/core/expect_column_stdev_to_be_between.py [hunk 9]
```diff
@@ -191,6 +198,27 @@ class ExpectColumnStdevToBeBetween(ColumnAggregateExpectation):
...
+                "const": model._library_metadata,
+            }
+            schema["properties"]["short_description"] = {
+                "type": "string",
+                "const": EXPECTATION_SHORT_DESCRIPTION,
+            }
+            schema["properties"]["supported_data_sources"] = {
+                "type": "array",
+                "const": SUPPORTED_DATA_SOURCES,
+            }
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_stdev_to_be_between.py]:
```suggestion
            schema["properties"]["metadata"]["properties"].update(
                {
                    "data_quality_issues": {
                        "title": "Data Quality Issues",
                        "type": "array",
                        "const": DATA_QUALITY_ISSUES,
                    },
                    "library_metadata": {
                        "title": "Library Metadata",
                        "type": "object",
                        "const": model._library_metadata,
                    },
                    "short_description": {
                        "title": "Short Description",
                        "type": "string",
                        "const": EXPECTATION_SHORT_DESCRIPTION,
                    },
                    "supported_data_sources": {
                        "title": "Supported Data Sources",
                        "type": "array",
                        "const": SUPPORTED_DATA_SOURCES,
                    },
                }
            )
```

File: great_expectations/expectations/core/expect_column_sum_to_be_between.py [hunk 10]
```diff
@@ -182,6 +189,27 @@ class ExpectColumnSumToBeBetween(ColumnAggregateExpectation):
...
+                "const": model._library_metadata,
+            }
+            schema["properties"]["short_description"] = {
+                "type": "string",
+                "const": EXPECTATION_SHORT_DESCRIPTION,
+            }
+            schema["properties"]["supported_data_sources"] = {
+                "type": "array",
+                "const": SUPPORTED_DATA_SOURCES,
+            }
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_sum_to_be_between.py]:
```suggestion
            schema["properties"]["metadata"]["properties"].update(
                {
                    "data_quality_issues": {
                        "title": "Data Quality Issues",
                        "type": "array",
                        "const": DATA_QUALITY_ISSUES,
                    },
                    "library_metadata": {
                        "title": "Library Metadata",
                        "type": "object",
                        "const": model._library_metadata,
                    },
                    "short_description": {
                        "title": "Short Description",
                        "type": "string",
                        "const": EXPECTATION_SHORT_DESCRIPTION,
                    },
                    "supported_data_sources": {
                        "title": "Supported Data Sources",
                        "type": "array",
                        "const": SUPPORTED_DATA_SOURCES,
                    },
                }
            )
```

File: great_expectations/expectations/core/expect_column_unique_value_count_to_be_between.py [hunk 11]
```diff
@@ -181,6 +204,29 @@ class ExpectColumnUniqueValueCountToBeBetween(ColumnAggregateExpectation):
...
+                "const": model._library_metadata,
+            }
+            schema["properties"]["short_description"] = {
+                "type": "string",
+                "const": EXPECTATION_SHORT_DESCRIPTION,
+            }
+            schema["properties"]["supported_data_sources"] = {
+                "type": "array",
+                "const": SUPPORTED_DATA_SOURCES,
+            }
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/expect_column_unique_value_count_to_be_between.py]:
```suggestion
            schema["properties"]["metadata"]["properties"].update(
                {
                    "data_quality_issues": {
                        "title": "Data Quality Issues",
                        "type": "array",
                        "const": DATA_QUALITY_ISSUES,
                    },
                    "library_metadata": {
                        "title": "Library Metadata",
                        "type": "object",
                        "const": model._library_metadata,
                    },
                    "short_description": {
                        "title": "Short Description",
                        "type": "string",
                        "const": EXPECTATION_SHORT_DESCRIPTION,
                    },
                    "supported_data_sources": {
                        "title": "Supported Data Sources",
                        "type": "array",
                        "const": SUPPORTED_DATA_SOURCES,
                    },
                }
            )
```

**Review by NathanFarmer — CHANGES_REQUESTED** [review]
Suggested changes and then `invoke schemas --sync`

**Review by NathanFarmer — APPROVED** [review]

**Pull request merged**